graf2d/cocoa/src/TGCocoa.mm

```diff
@@ -11,13 +11,22 @@
 
 //#define NDEBUG
 
-#include "TGCocoa.h"
+#ifdef WIN32
```

Comment:
I don't understand what Windows has to do with Cocoa?

Reply:
I was talking with Bertand, I will check if it is still needed (it was a part of TGLInclude.h file)